onnx/common/common.h

```diff
@@ -2,7 +2,30 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
 #pragma once
 
 #define ONNX_UNUSED_PARAMETER(x) (void)(x)
+
+#ifdef ONNX_NO_EXCEPTIONS
+#define ONNX_THROW(...)                                 \
+  std::cerr << ONNX_NAMESPACE::MakeString(__VA_ARGS__); \
```

Comment:
Add ```do { ... } while (false)``` ?

Reply:
I am unclear why this is marked resolved. Wouldn't this macro fail in a usage like ```if (cond) ONNX_THROW(...)```?